cmd/rest/main.go

```diff
@@ -20,6 +20,11 @@ func main() {
 // Run will bind our layers all together
 func Run() error {
 	// TODO: Handle errors, migration, configs ...
+	err := cfg.LoadEnvVar()
+	if err != nil {
+		return err
+	}
+
 	opt := cfg.NewOptions()
```

Comment:
Honestly, i don't really see a sense of calling config as options. 
Config is pretty meaningful name.

Reply:
Yes, unequivocally. GetConfig is now there.